hypha/apply/determinations/templatetags/determination_tags.py

```diff
@@ -12,3 +12,8 @@ def show_determination_button(user, submission):
         return can_edit_determination(user, submission.determinations.active(), submission)
     except ObjectDoesNotExist:
         return can_create_determination(user, submission)
+
+
+@register.filter
+def has_draft_determination(user, submission):
```

Comment:
See comment above `show_determination_button` should be enough to identify ability to edit.

Reply:
Removed this and updated determination_button.html to use `submission.determinations.last.is_draft` instead